setup.py

```diff
@@ -145,7 +145,9 @@ def build_module(self, module, module_file, package):
                 'urllib3.contrib._securetransport', 'urllib3.util',
                 'urllib3._async', 'urllib3._backends',
                 ],
-      requires=[],
+      # We'll probably want to vendor this for a real release, but for now
+      # this works.
+      requires=["h11"],
```

Comment:
I see that `install_requires` from setuptools is [recommended in the PyPA sample project](https://github.com/pypa/sampleproject/blob/4ac29396ad27661533608175f7b0c1058df86cf3/setup.py#L119-L125) instead of `requires` from distutils. And indeed, without changing `requires` to `install_requires`, `pip install -e .` does not install h11.

Reply:
Wow, yeah, you're absolutely right. I understand how distutils/setuptools got this way, but I still can't wait until we can tear them down and switch to something else.